vsintegration/src/FSharp.Editor/Options/EditorOptions.fs

```diff
@@ -42,6 +41,11 @@ type LanguageServicePerformanceOptions =
     { EnableInMemoryCrossProjectReferences: bool
       ProjectCheckCacheSize: int }
 
+[<CLIMutable>]
+type AdvancedOptions =
+    { IsBlockStructureEnabled: bool 
+      IsOutliningEnabled: bool }
+
 [<Export(typeof<ISettings>)>]
 type internal Settings [<ImportingConstructor>](store: SettingsStore) =
```

Comment:
This is where you access the data in our settings store, with default values listed below